docs/history/changelog.md

```diff
@@ -202,6 +202,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
   * [#1353](https://github.com/open-contracting/standard/pull/1353) Replace "tender submissions" with "bid submissions".
   * [#1401](https://github.com/open-contracting/standard/pull/1401) Replace "total value" with "value".
   * [#1403](https://github.com/open-contracting/standard/pull/1403) Replace "contract notice" with "contract signature notice".
+  * [](https://github.com/open-contracting/standard/pull/1487) `planning.budget.project`, to remove no translation option rationale.
```

Comment:
```suggestion
  * [#1487](https://github.com/open-contracting/standard/pull/1487) Remove sentence about translation options from `planning.budget.project`.
```

Reply:
Good catch

```
`planning.budget.project`, to remove sentence about translation options.
```

Reply:
And maybe I should move it below (or above?) that one to keep all the ones related to specific fields together?

Reply:
Yes, good call